src/kibana/components/vislib/visualizations/line_chart.js

```diff
@@ -73,6 +73,17 @@ define(function (require) {
       var touchableRadius = 12;
       var tooltip = this.tooltip;
       var isTooltip = this._attr.addTooltip;
+      var radii = _(data)
+        .map(function (series) { return _.map(series, function (point) { return point._input.z; }); })
+        .flatten()
+        .value();
+
+      radii = {
+        min: _.min(radii),
+        max: _.max(radii)
+      };
+
+      var radiusStep = ((radii.max - radii.min) || (radii.max * 100)) / 20;
```

Comment:
You should just need to vary the `20` on this line to change the ratio between the min and max dots. A bigger number here should result in a bigger difference between the smallest dots and the biggest ones